src/main/java/org/agbrothers/englishbot/buttonsbuilder/MainMenuButtonsBuilder.java

```diff
@@ -16,19 +17,19 @@
         Map<String, String> keyboardMap;
         switch (messageText) {
             case LESSONS:
-                keyboardMap = new HashMap<>();
+                keyboardMap = new LinkedHashMap<>();
```

Comment:
Could you please use this map creation to the variable initialization at line 17?